Ex04.Menus.Test/Program.cs

```diff
@@ -1,12 +1,38 @@
 ﻿using System;
+using B23_Ex04_Ronen_319047718_Ido_315942193;
 
 namespace Ex04.Menus.Test
 {
     public class Program
     {
         public static void Main()
         {
+            const bool v_IsMainMenu = true;
```

Comment:
only one filed is relevant
if its not main so it clearly no Main

Reply:
Got it no problem